src/oscar/apps/dashboard/views.py

```diff
@@ -189,6 +216,9 @@ def get_stats(self):
         if user.is_staff:
             stats.update(
                 total_site_offers=self.get_active_site_offers().count(),
+                total_voucher_offers=self.get_active_voucher_offers().count(),
```

Comment:
I think it should be possible to fetch all three counts in one query with aggregation, something like

```
ConditionalOffer.objects.filter(end_datetime__gt=now()).values('offer_type').aggregate(count=Count('id'))
```